bin/ghe-restore

```diff
@@ -97,6 +97,14 @@ elif [ "$GHE_VERSION_MAJOR" -ge 2 ]; then
     restore_settings=true
 fi
 
+# If you restore to a server in a replication pair
+if ghe-ssh "$GHE_HOSTNAME" -- "ghe-repl-status -r 2>/dev/null" \
+  | grep -E "replica|primary"; then
```

Comment:
You'll want to `>/dev/null` the `grep` output here or we'll get a matching line on stdout. Outside of that, I'm good to merge.